debezium-connector-yugabytedb2/pom.xml

```diff
@@ -41,12 +41,12 @@
         <dependency>
             <groupId>org.yb</groupId>
             <artifactId>yb-client</artifactId>
-            <version>0.8.17-SNAPSHOT</version>
+            <version>0.8.15-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.yb</groupId>
             <artifactId>yb-client</artifactId>
-            <version>0.8.17-SNAPSHOT</version>
+            <version>0.8.15-SNAPSHOT</version>
```

Comment:
Using this version number, create a release for `2.12.4.1-b2` and another release for the latest master of `yugabyte-db`